modules/react/collection/stories/mdx/examples/OverflowVerticalList.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import {ActionBar, useActionBarModel} from '@workday/canvas-kit-react/action-bar';
+import {PrimaryButton} from '@workday/canvas-kit-react/button';
+import {Box} from '@workday/canvas-kit-react/layout';
+import styled from '@emotion/styled';
+import {StyledType} from '@workday/canvas-kit-react/common';
+
+type MyActionItem = {
+  id: string;
+  text: React.ReactNode;
+};
+
+const StyledActionbarList = styled(ActionBar.List)<StyledType>({
+  '> *': {
+    flex: '0 0 auto',
+  },
+});
```

Comment:
This uses style props so using this for now, we can always clean up the example later